### ISSUE TITLE
Player changed today?

### ISSUE THREAD

**Issue by SuperSajuuk** [YePpHa/YouTubeCenter#1747]
I just noticed the whole player has changed to a different format. Here's a screenshot showing that new format. The one thing I noticed is that 40px are lost from the video height due to the expanded player controls, meaning that setting the player to 1280x720 will actually created 1280x680 player, clearly out of proportion.

It's an interesting player, but does anyone know how to get rid of it and use the older format? Or is this what we have to deal with now?

[![Gyazo](http://i.gyazo.com/eddc8546285c08ea3dfb3ae39c702e61.png)](http://gyazo.com/eddc8546285c08ea3dfb3ae39c702e61)

**Comment by LazyBoot** [comment]
It does seem like YouTube has started rolling out a new player to some people... But if I leave youtube center (build 507) active I get this:
![Screenshot](http://i.imgur.com/3wqidWk.jpg)

**Comment by Yonezpt** [comment]
Both of you are getting the transparent player which is still incomplete and is most likely having conflicts with YTC. If you want to opt-out you can do it via this link: https://www.youtube.com/testtube

And click on the "Restore original player" button.

**Comment by hamkah2c** [comment]
Errmm, I can't see/found the "restore original player" button.
![image](https://cloud.githubusercontent.com/assets/8862875/7495451/3fca209a-f438-11e4-9f75-5a89d74a21de.png)

**Comment by Zarggg** [comment]
Try deleting all cookies set by YouTube. In the past YouTube has used cookies to track whether a user has an experimental player or layout.

In Chrome, you can do this by going to Settings -> Privacy -> Content settings... -> Cookies -> All cookies and site data... and searching for "youtube.com".

**Comment by Yonezpt** [comment]
@Zarggg You can't get rid of it via the cookies either? Can't believe they are forcing a feature that is still under testing.

**Comment by Zarggg** [comment]
No, removing cookies *did* restore the regular player for me. I just thought it was amusing/ironic that I got the new one shortly after posting on this issue.

**Comment by SuperSajuuk** [comment]
I believe the cookie that YT uses to track people is the VISITOR_INFO1_LIVE cookie. Simply deleting that one will remove any tracking data and give you a plain cookie.

The new player just sucks. It's like they think I'm blind and that I won't notice whether a video is paused or not >_>
